libafl/src/monitors/stats/mod.rs

```diff
@@ -118,21 +118,29 @@ pub struct ItemGeometry {
     pub own_finds: u64,
     /// How much entries were imported
     pub imported: u64,
-    /// The stability, stringified
-    pub stability: String,
+    /// The stability in percent, ranges from 0 to 100.
```

Comment:
Shouldn't it be a float from 0 to 1?

Reply:
I mean probably it wouldn't hurt? At least one or two more decimals might be useful